app/src/test/java/com/codesoom/assignment/domain/ProductTest.java

```diff
@@ -0,0 +1,32 @@
+package com.codesoom.assignment.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("Product 클래스")
+class ProductTest {
+    final Long ID = 0L;
+    final String NAME = "My Toy";
+    final String MAKER = "My Home";
+    final Long PRICE = 5000L;
+    final String IMAGE_URL = "https://cdn.pixabay.com/photo/2016/10/01/20/54/mouse-1708347_1280.jpg";
+
+    @DisplayName("Product가 정상적으로 생성되고 값이 세팅되었는지 확인하는 테스트")
```

Comment:
테스트에 대한 설명에 `테스트`가 들어가는 것은 아쉬운 일입니다. 다음과 같이 하면 어떨까요?
추가로 좀 더 자세한 설명을 주석으로 남겨보는 것도 의미있는 일일 것 같아요.
```suggestion
    @DisplayName("Product가 정상적으로 생성되고 값이 세팅되었는지 확인합니다")
```

Reply:
`테스트` 라는 사실이 들어가면 오히려 좋지 않군요 .. !
다만 더 자세한 설명을 남겨보려고 해봤는데 사실 어떤 설명을 적어줘야 할지 많은 고민이 되었습니다..
혹시 참고해볼만한 샘플이 있을까요 ???

Reply:
사실 샘플이라 할 만한 것이 딱히 없습니다. 회사, 팀마다 스타일이 다르기 때문에 최대한 많이 열심히 작성해보며 자신만의 스타일을 찾는 것이 중요하다고 생각해요. 한편 제가 이야기하는 것들은 제 스타일이기도 합니다.